### DISCUSSION TITLE
Consider publishing development/contributing guides to docs.renovatebot.com

### DISCUSSION THREAD

**Discussion by rarkins** [renovatebot/renovate#25172] [Suggest an Idea]
### Tell us more.

Advantages:
- They will be more user-friendly for contributors versus today's purely markdown format

Disadvantages:
- More work for us to maintain
- Most users of the docs site do not contribute, and their search results may find contributing docs which confuse them

**Comment by viceice** [comment]
So you suggest to no longer publish the mkdocs generated files?

**Reply by rarkins** [reply]
I mean to discuss whether we should including docs/development/* into docs.renovatebot.com

**Reply by cgrindel** [reply]
Personally, I would think that a link on the docs.renovatebot.com website to the markdown docs would be sufficient. I assume that this came up to make the docs more visible. You could even dedicate a page on the docs.renovatebot.com website for dev docs and have a single link. People just need to find them.

**Reply by HonkingGoose** [reply]
I like @cgrindel's idea best, because it doesn't affect our search too much. And we'll have a nice "How to contribute page" in the hosted docs, with all the relevant links in one place.

**Reply by HonkingGoose** [reply]
Here's our new "Contributing to Renovate" page:

- #25238
